astropy/modeling/models.py

```diff
@@ -57,16 +57,43 @@
 
 
 MODELS_WITH_CONSTRAINTS = [
-    AiryDisk2D, Moffat1D, Moffat2D, Box1D, Box2D,  # noqa: F405
-    Const1D, Const2D, Ellipse2D, Disk2D,  # noqa: F405
-    Gaussian1D, Gaussian2D,  # noqa: F405
-    Linear1D, Lorentz1D, RickerWavelet1D, RickerWavelet2D,  # noqa: F405
-    PowerLaw1D, Sersic1D, Sersic2D,  # noqa: F405
-    Sine1D, Cosine1D, Tangent1D, ArcSine1D, ArcCosine1D, ArcTangent1D,  # noqa: F405
-    Trapezoid1D, TrapezoidDisk2D,  # noqa: F405
-    Chebyshev1D, Chebyshev2D, Hermite1D, Hermite2D, Legendre2D, Legendre1D,  # noqa: F405
-    Polynomial1D, Polynomial2D, Voigt1D, KingProjectedAnalytic1D,  # noqa: F405
-    NFW  # noqa: F405
+    AiryDisk2D,  # noqa: F405
+    Moffat1D,  # noqa: F405
+    Moffat2D,  # noqa: F405
+    Box1D,  # noqa: F405
+    Box2D,  # noqa: F405
+    Const1D,  # noqa: F405
+    Const2D,  # noqa: F405
+    Ellipse2D,  # noqa: F405
+    Disk2D,  # noqa: F405
+    Gaussian1D,  # noqa: F405
+    Gaussian2D,  # noqa: F405
+    Linear1D,  # noqa: F405
+    Lorentz1D,  # noqa: F405
+    RickerWavelet1D,  # noqa: F405
+    RickerWavelet2D,  # noqa: F405
+    PowerLaw1D,  # noqa: F405
+    Sersic1D,  # noqa: F405
+    Sersic2D,  # noqa: F405
+    Sine1D,  # noqa: F405
+    Cosine1D,  # noqa: F405
+    Tangent1D,  # noqa: F405
+    ArcSine1D,  # noqa: F405
+    ArcCosine1D,  # noqa: F405
+    ArcTangent1D,  # noqa: F405
+    Trapezoid1D,  # noqa: F405
+    TrapezoidDisk2D,  # noqa: F405
+    Chebyshev1D,  # noqa: F405
+    Chebyshev2D,  # noqa: F405
+    Hermite1D,  # noqa: F405
+    Hermite2D,  # noqa: F405
+    Legendre2D,  # noqa: F405
+    Legendre1D,  # noqa: F405
+    Polynomial1D,  # noqa: F405
+    Polynomial2D,  # noqa: F405
+    Voigt1D,  # noqa: F405
+    KingProjectedAnalytic1D,  # noqa: F405
+    NFW,  # noqa: F405
```

Comment:
I am not super thrilled with having to suppress flake8 on every line of this. Does anyone have any suggestions?

Reply:
Explicit import?

Reply:
Suppressed it in the `setup.cfg` for now.

It does need an explicit import, but since this is part of the main modeling API for getting models I would rather not risk modifying it for now. Issue #13981 will track this issue for now.